inst/scripts/orderly_run.R

```diff
@@ -0,0 +1,12 @@
+## Prepare params
+args <- readRDS(Sys.getenv("ORDERLY_RUN_ARGS_PATH"))
```

Comment:
I think a short comment explaining that this is the easiest way of parametrising a job might help us in future

Reply:
Thanks, added some notes to that.